### DISCUSSION TITLE
Is there a way to implement JINT in .NET WebBrowser control?

### DISCUSSION THREAD

**Discussion by scarletquasar** [sebastienros/jint#970] [Q&A] [answered]
I'm building a project and I would like to know if there is how and how to apply Jint to the control, I researched but I didn't find a concrete answer

**Comment by lahma** [comment, marked as answer]
I think that would be quite hard. I haven't worked with WebBrowser, but I doubt it would allow such integration.

**Reply by scarletquasar** [reply]
No problem then, i was looking for something like that to integrate in my runtime project. Using old ie JavaScript syntax is not good, ill write an adapter to transform jint responses in javascript actions and inject in the webbrowser.
